content/en/docs/concepts/configuration/pod-priority-preemption.md

```diff
@@ -67,19 +67,32 @@ is disabled, you cannot set `priorityClassName` in new Pods.
 
 ## How to disable preemption
 
-{{< note >}}
-**Note**: In Kubernetes 1.11, critical pods (except DaemonSet pods, which are
-still scheduled by the DaemonSet controller) rely on scheduler preemption to be
-scheduled when a cluster is under resource pressure. For this reason, we do not
-recommend disabling this feature. If you still have to disable this feature,
-follow the instructions below.
-{{< /note >}}
+{{< note >}} **Note**: In Kubernetes 1.11, critical pods (except DaemonSet pods,
+which are still scheduled by the DaemonSet controller) rely on scheduler
+preemption to be scheduled when a cluster is under resource pressure. For this
+reason, you will need to run an older version of Rescheduler if you decide to
+disable preemption. More on this is provided below. {{< /note >}}
+
+#### Option 1: Disable both Pod priority and preemption
+
+Disabling Pod priority disables preemption as well. In order to disable Pod
+Priority, set the feature to false for API server, Scheduler, and Kubelet.
```

Comment:
Can you please elaborate on disabling feature for scheduler and kubelet?

Reply:
Not sure what you have in mind, but instructions are given below which is setting the feature gate flag.

Reply:
That is right, I missed that part, re: scheduler, we can add something similar like `--feature-gates..` but if we include that we should warn on using disablePreemption flag

Reply:
Because people may unset disablePreemption flag and expect preemption to happen when priority has been disabled. IIRC both of them have to be set.